src/Reader/Extension/Thread/Entry.php

```diff
@@ -15,9 +15,9 @@ class Entry extends Extension\AbstractEntry
     /**
      * Get the "in-reply-to" value
      *
-     * @return string
+     * @return void
      */
-    public function getInReplyTo()
+    public function getInReplyTo(): void
```

Comment:
Please do not add php typehints as this will is a BC break unless the class is final.